examples/get_started/03_figure_element.py

```diff
@@ -0,0 +1,80 @@
+"""
+3. Figure elements
+==================
+The figure shows the naming of figure elements in PyGMT.
+
+- :meth:`pygmt.Figure()`: having a number of plotting methods. Every new
+  figure must start with the creation of a :meth:`pygmt.Figure()` instance
+- ``frame``: setting plot or map boundaries (**WSNE**, **wsne**, or
+  **lbtr**), adding annotations, ticks, gridlines (**afg**), axis labels
+  (**+l**), and title (**+t**), e.g., in :meth:`pygmt.Figure.basemap`.
+  Detailed examples can be found at `frame and axes attributes
+  <https://www.pygmt.org/latest/tutorials/basics/frames.html>`_.
```

Comment:
Use relative intersphinx link - https://docs.readthedocs.io/en/stable/guides/intersphinx.html#using-intersphinx
```suggestion
  Detailed examples can be found at :doc:`frame and axes attributes
  </tutorials/basics/frames>`.
```